tests/Jackalope/PropertyTest.php

```diff
@@ -2,8 +2,67 @@
 
 namespace Jackalope;
 
+use PHPCR\PropertyType;
+
 class PropertyTest extends TestCase
 {
+    public function provideGetNode()
+    {
+        return array(
+            array(
+                array('1234-1234', '4321-4321'),
+                array(),
+                'Internal Error: Could not find a referenced node. If the referencing node is a frozen version, this can happen, otherwise it would be a bug.',
+            ),
+            array(
+                array('1234-1234', '4321-4321'),
+                array('1234-1234'),
+                'Internal Error: Could not find a referenced node. If the referencing node is a frozen version, this can happen, otherwise it would be a bug.',
+            ),
+            array(
+                array('1234-1234', '4321-4321'),
+                array('1234-1234', '4321-4321'),
+            ),
+            array(
+                array('1234-1234', '4321-4321'),
+                array('4321-4321', '1234-1234'),
+                'Internal Error: Could not find a referenced node. If the referencing node is a frozen version, this can happen, otherwise it would be a bug.',
```

Comment:
hm, the mystery deepends. when backporting this to master branch, i had to remove this line again as the order no longer seems to matter. #277 keeps this message removed and tests succeed.

that line 450 would make no sense as the getNodesByIdentifiers eliminates duplicate ids. Property::getReferencedNodes should restore the corect order however. do things work for you now with the latest 1.1 branch?